scripts/ci/publish.ts

```diff
@@ -657,9 +657,7 @@ Check them out at https://github.com/prisma/ecosystem-tests/actions?query=workfl
 async function getEnginesCommitHash(): Promise<string> {
   const prismaPath = path.resolve(process.cwd(), './packages/engines/package.json')
   const pkg = JSON.parse(await fs.promises.readFile(prismaPath, 'utf-8'))
-  // const engineVersion = pkg.prisma.version
```

Comment:
Any idea why this was commented out and not used in the first place?

Reply:
Ah, your suggestion actually goes into the `@prisma/engines-version` package.json and then gets `prisma.enginesVersion` there?

Reply:
Yes, that's the best way to do this. We put the hash in the package.json

Reply:
The comment is about a different hash, not the one we are interested here